titus-ext/elasticsearch/src/main/java/com/netflix/titus/ext/elasticsearch/ElasticsearchTaskDocumentPublisher.java

```diff
@@ -66,13 +70,20 @@
     private final ObjectMapper objectMapper;
     private final SimpleDateFormat indexDateFormat;
     private final SimpleDateFormat taskDateFormat;
+    private final AtomicInteger docsToBePublished = new AtomicInteger(0);
+    private final AtomicInteger docsPublished = new AtomicInteger(0);
+    private final AtomicInteger errorJsonConversion = new AtomicInteger(0);
+    private final AtomicInteger errorEsClient = new AtomicInteger(0);
+    private final AtomicInteger errorInPublishing = new AtomicInteger(0);
+    private final AtomicInteger isAlive = new AtomicInteger(0);
```

Comment:
I suggest you use the `timeSinceLastPublish` strategy to where you are publishing a metric that denotes how long it has been since the last publish such that alerting on it will be easier.